apps/settings/js/panels/operator_settings/models/operator_manager.js

```diff
@@ -142,21 +142,22 @@ define(function(require) {
 
     this.debug('search');
     this._searching = true;
-    return this._connWrapper.search().then((networks = []) => {
-      var operators = networks.map((network) => OperatorItem(network));
-      operators.forEach((operator) => {
+    return this._connWrapper.search().then(function(networks = []) {
```

Comment:
ooh seems default argument with arrow cannot be recongnized in current jsdoc `networks = []) =>`

please use 
```
then((networks) => {
networks = networks || [];
var operators =....
```